Retarus/pyproject.toml

```
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "retarus"
+version = "0.1.0"
+description = "Retarus intake module"
+authors = ["Alexandre Léonardi <alexandre.leonardi@sekoia.io>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.12"
+sekoia-automation-sdk = "^1.11.0"
+orjson = "^3.7.7"
+websocket-client = "^1.3.3"
+python-dateutil = "^2.8.2"
+prometheus-client = "^0.16.0"
```

```suggestion
prometheus-client = "^0.19.0"
```

You can update to the latest release.